### PR TITLE
Update chunk names to all chunks and small text changes

### PR DESCRIPTION
**Pull request by celinech**
Almost all files are modified!
Minor impact on the pipeline ==> saved figures saved by the execution could be found under a new name.

### CODE REVIEW

File: 04_additionalControls.Rmd [hunk 1]
```diff
@@ -181,12 +181,12 @@ fltd@meta.data <- tibble::column_to_rownames(fltd@meta.data, "Row.names")
...
 ## Number of heterotypic doublets estimation (nExp parameter)
 
-```{r homotypic-proportion}
+```{r heterotypic-proportion}
 if(atlas){
   annotations <- fltd@meta.data[, "celltype_DF"]
 } else {
   annotations <- fltd@meta.data[, paste0("RNA_snn_res.",clust_res)]
-}
+}ssssssss
```
Comment (Rumengol) [inline 04_additionalControls.Rmd:189]: ?

**Review by Rumengol — CHANGES_REQUESTED** [review]
Overall I agree with the new names, however we should adopt a single naming convention, as we currently have three, sometimes overlapping in the same name:
- `kebab-case`
- `snake_case`
- `camelCase`

I would argue in favor of the snake_case, as it is both more readable than the camelCase and it less error prone than the kebab-case.

**Comment by celinech** [comment]
I agree that I don't fit a single format.

1. However, I remember I've read that kebab-case is better suited for chunk names. I'm sorry, I can't find the source of it. And as the majority of the chunk names are in kebab-case, I would prefer to go for it, to not spend too much time on it.

2. Would you like to change these ?
    - get-topNmarkers, cluster-dimPlot-markers, cluster-dotPlot-markers, cluster-dotPlot-goi
topNmarkers is already a variable in the code.
DimPlot, FeaturePlot, ... are pre-existing functions from seurat.

**Comment by Rumengol** [comment]
Let's go with kebab-case then, I don't see the logic but since it's for file names it shouldn't be an issue.

For references to existing variables/functions, I'd say we can keep them as they are.

**Review by Rumengol — APPROVED** [review]

**Pull request merged**